index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
+<em>"compatible with the Verifiable Credentials ecosystem"</em>, and the result
+of the transformation, which is a <a>conforming document</a>, is a
+<a>verifiable credential</a>. Specifications that contain transformations to the
+data model described by this specification:
+        </p>
+
+        <ul>
+          <li>
+MUST identify if the transformation to this data model is uni-directional or
+bi-directional.
+          </li>
+          <li>
+MUST preserve the `@context` values when performing bi-directional
+transformation.
+          </li>
+          <li>
+MUST result in a <a>conforming document</a>.
+          </li>
+          <li>
+MUST specify a media type for the input document, which will be transformed into
```

Comment:
I really like this one, because it places at least some constraint on "weird things people invent" being mapped to `vc+ld+json`... it raises the quality bar, and it requires coordination with IETF, in order to ensure interop, although I assume this allows for "unregistered" media types?

Since this is normative, should the media type be required to be registered? or is it ok for people to just make up strings here?

Reply:
I would prefer registered media type

Reply:
@mprorock this, along w/ a pass to implement other feedback based on the special topic call, was done in 08ad96a0b98540c6b01257d10e1b3faf22c07fd9